pkg/operator/controllers/workaround/systemreserved.go

```diff
@@ -45,7 +46,10 @@ func (sr *systemreserved) Name() string {
 	return "SystemReserved fix for bz-1857446"
 }
 
-func (sr *systemreserved) IsRequired(clusterVersion *version.Version) bool {
+func (sr *systemreserved) IsRequired(clusterVersion *version.Version, cluster *arov1alpha1.Cluster) bool {
+	if cluster.Spec.Features.ReconcileAutoSizedNodes {
+		return false
+	}
```

Comment:
For visibility. We have discussed the need for manual enable/disable because there is a need to test in a small region first.
Can update later on and make default disabled for `OCP >= 4.8`.

WorkItem: https://msazure.visualstudio.com/AzureRedHatOpenShift/_workitems/edit/12736889